.bettercodehub.yml

```
@@ -0,0 +1,19 @@
+component_depth: 1
```

```suggestion
component_depth: 1
default_excludes: true
```

Is there a reason not to include the default excludes?

---

Where did you find `default_excludes`? It's not in [their configuration documentation](https://bettercodehub.com/docs/configuration-manual). I can imagine that it might be helpful, but I don't know what it does.

---

It was in the default config of this repo.

---

I'm lost. Which repo? There was no default BetterCodeHub config for the 3601-iteration-template repo, and I can't find any evidence that BetterCodeHub supports a `default_excludes` entry in their config file.

---

If you go to the bettercodehub site and open the settings for the repo it shows the current config.

---

Yup. I totally see it now. I have no idea why that's not in their docs on the form of these files. I'll definitely add that.